src/executables/csql.c

```diff
@@ -1460,6 +1473,21 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	}
 
       break;
+
+    case S_CMD_CONNECT:
+      if (csql_arg->sysadm != true)
+	{
+	  error_code = csql_connect ((argument[0] == '\0') ? NULL : argument, csql_arg);
+	  if (error_code != NO_ERROR)
+	    {
+	      return error_code;
```

Comment:
I think it's better to give warning to user only when ';connect' is failed.

Reply:
I think so. Thank you!